webextensions/api/captivePortal.json

```diff
@@ -0,0 +1,96 @@
+{
+  "webextensions": {
+    "api": {
+      "captivePortal": {
+        "getState": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/captivePortal/getState",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "68"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "getLastChecked": {
```

Comment:
It's good practice I think to order features, so this should come before `getState`.

Reply:
Thanks, noted and done